### PR TITLE
fix structure sql query on _get_invoice_matching_query

### PR DESCRIPTION
**Pull request by duybq86**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by duong77476-viindoo** [comment]
Odoo 14 is no longer support, you should make a pr at your fork

File: addons/account/models/account_reconcile_model.py [hunk 1]
```diff
@@ -749,13 +811,14 @@ def _get_invoice_matching_query(self, st_lines_with_partner, excluded_ids):
...
-            query += "AND aml.date >= %(aml_date_limit)s"
+            date_limit = fields.Date.context_today(
+                self) - relativedelta(months=self.past_months_limit)
+            query += " AND aml.date >= %(aml_date_limit)s"
             params['aml_date_limit'] = date_limit
 
         # Filter out excluded account.move.line.
         if excluded_ids:
-            query += 'AND aml.id NOT IN %(excluded_aml_ids)s'
+            query += ' AND aml.id NOT IN %(excluded_aml_ids)s'
```
Comment (MiquelRForgeFlow) [inline addons/account/models/account_reconcile_model.py:821]: Only this change is acceptable. I think you shouldn't run pre-commit/black, as it difficult to find the relevant change, so I think you should revert the rest of changes. Also, you should point to v15, as the oldest supported stable version.

Comment (duybq86) [reply addons/account/models/account_reconcile_model.py:821]: thank so much
because i run odoo ver 14, also i want fix some issue on this branch

**Comment by duybq86** [comment]
close merge and no code formatting, just fix the problem

**Pull request closed without merging**